cmd/kubeadm/app/apis/kubeadm/v1beta1/bootstraptokenstring.go

```diff
@@ -26,6 +26,7 @@ import (
 	bootstraputil "k8s.io/cluster-bootstrap/token/util"
 )
 
+// DEPRECATED - This group version of BootstrapTokenString is deprecated by apis/kubeadm/v1beta2/BootstrapTokenString.
```

Comment:
not sure why this PR includes these changes.

Reply:
I try to use git rebase, It cause some previous commit are reappear on this pr;